### PR TITLE
Removed imports to db module where unnecessary

### PR DESCRIPTION
**Pull request by psav**
Purpose or Intent
=================

Removing some 5.5 supported functions and trying to cleanup some imports to db

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 408bf139272c89b44b437615e796f7f5b5aee2dc:

`utils/db.py`:
- :red_circle: Line 4:1: **F401** *'NamedTemporaryFile' imported but unused*
- :red_circle: Line 6:1: **F401** *'yaml' imported but unused*
- :red_circle: Line 16:1: **F401** *'load_data_file' imported but unused*
- :red_circle: Line 18:1: **F401** *'data_path' imported but unused*
- :red_circle: Line 19:1: **F401** *'SSHClient' imported but unused*

Please, rectify these issues :smirk: .

Commit inspection report:
**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 306db6e4cfece5f9817e6355d56428193c37ade2:

Everything seems all right from lint perspective :smile: .

Commit inspection report:
**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ef67043958a9a7addcb16972e83c6a7ade00285b:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: utils/appliance/__init__.py [hunk 1]
```diff
@@ -1540,64 +1540,54 @@ def zone_description(self):
...
+            ruby_template = data_path.join('utils', 'cfmedb_set_config.rbt')
+            ruby_replacements = {
+                'config_file': dest_yaml
+            }
+            temp_ruby = load_data_file(ruby_template.strpath, ruby_replacements)
+            self.ssh_client.put_file(temp_ruby.name, dest_ruby)
+
+            # Run it
+            result = self.ssh_client.run_rails_command(dest_ruby)
+            if not result.rc:
```
Comment (mfalesni) [inline utils/appliance/__init__.py]: Man, you can do this:
```python
if self.ssh_client.run_rails_command(dest_ruby):
    fire(...)
else:
    raise ...
```

Comment (mfalesni) [inline utils/appliance/__init__.py]: Though this is just code moving :)

**Pull request merged**